domain/src/main/java/com/example/domain/Item.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.domain
+
+data class Item(
+    val header: Header,
+    var body: Body
+) {
+    fun priceString(price: Float): String {
```

Comment:
I like this approach where we have this function within the `Item` class instead of doing the formatting outside (like in activities or fragments), since it's only formatting the String right. Reason is this approach improves code reuse, and prevents and code duplication.

We can improve this further by moving this function inside the `Base` class, since that is the class holding the `price` field.